kv/kvserver: TestFlowControlAdmissionPostSplitMergeV2 failed

kv/kvserver.TestFlowControlAdmissionPostSplitMergeV2 [failed](https://mesolite.cluster.engflow.com/invocations/default/540c7c78-10ef-4efd-8bd2-701372c19cb2?testReportRun=3&testReportShard=5&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZzZXJ2ZXI6a3ZzZXJ2ZXJfdGVzdA==) on release-25.1.9-rc @ [2fcef9301ac20e8cb084cf7e2c2c24bede0ee1e7](https://github.com/cockroachdb/cockroach/commits/2fcef9301ac20e8cb084cf7e2c2c24bede0ee1e7):


```
=== RUN   TestFlowControlAdmissionPostSplitMergeV2
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestFlowControlAdmissionPostSplitMergeV22463735938
    test_log_scope.go:76: use -show-logs to present logs inline
    flow_control_integration_test.go:544: -- test log scope end --
test logs left over in: outputs.zip/logTestFlowControlAdmissionPostSplitMergeV22463735938
--- FAIL: TestFlowControlAdmissionPostSplitMergeV2 (20.48s)
=== RUN   TestFlowControlAdmissionPostSplitMergeV2/v2_enabled_when_leader_level=2
    --- FAIL: TestFlowControlAdmissionPostSplitMergeV2/v2_enabled_when_leader_level=2 (11.20s)
=== RUN   TestFlowControlAdmissionPostSplitMergeV2/v2_enabled_when_leader_level=2/kvadmission.flow_control.mode=apply_to_all
    datadriven.go:144: 
        /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/kv/kvserver/kvserver_test_/kvserver_test.runfiles/com_github_cockroachdb_cockroach/pkg/kv/kvserver/testdata/flow_control_integration_v2/admission_post_split_merge_v2_encoding_apply_to_all:1:
         
        output didn't match expected:
        @@ -84,11 +84,11 @@
           kvflowcontrol.tokens.eval.regular.unaccounted                     | 0 B      
           kvflowcontrol.tokens.send.elastic.available                       | 3.0 MiB  
           kvflowcontrol.tokens.send.elastic.deducted                        | 21 MiB   
           kvflowcontrol.tokens.send.elastic.deducted.force_flush_send_queue | 0 B      
           kvflowcontrol.tokens.send.elastic.deducted.prevent_send_queue     | 0 B      
        -  kvflowcontrol.tokens.send.elastic.returned                        | 0 B      
        +  kvflowcontrol.tokens.send.elastic.returned                        | 8.0 KiB  
           kvflowcontrol.tokens.send.elastic.returned.disconnect             | 0 B      
           kvflowcontrol.tokens.send.elastic.unaccounted                     | 0 B      
           kvflowcontrol.tokens.send.regular.available                       | 27 MiB   
           kvflowcontrol.tokens.send.regular.deducted                        | 21 MiB   
           kvflowcontrol.tokens.send.regular.deducted.prevent_send_queue     | 0 B      
        --- FAIL: TestFlowControlAdmissionPostSplitMergeV2/v2_enabled_when_leader_level=2/kvadmission.flow_control.mode=apply_to_all (6.64s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=5</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFlowControlAdmissionPostSplitMergeV2.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-52953

--- a/pkg/kv/kvserver/flow_control_integration_test.go
+++ b/pkg/kv/kvserver/flow_control_integration_test.go
@@ -381,6 +381,7 @@ func TestFlowControlBlockedAdmissionV2(t *testing.T) {
 func TestFlowControlAdmissionPostSplitMergeV2(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
+	skip.UnderDuressWithIssue(t, 150840)
 	defer setRACv2DebugVModule(t)()
 
 	testutils.RunValues(t, "kvadmission.flow_control.mode", []kvflowcontrol.ModeT{

